packages/patternfly-4/react-core/src/components/ClipboardCopy/ClipboardCopyButton.tsx

```diff
@@ -0,0 +1,53 @@
+import * as React from 'react';
+import styles from '@patternfly/patternfly/components/ClipboardCopy/clipboard-copy.css';
```

Comment:
Change to `@patternfly/react-styles/css/components/ClipboardCopy/clipboard-copy`